.github/workflows/hermes_open_project_recommendation.yaml

```diff
@@ -38,6 +38,10 @@ jobs:
           GOOGLE_APPLICATION_SERVICE_ACCOUNT: ${{ secrets.GOOGLE_APPLICATION_SERVICE_ACCOUNT }}
           TELEGRAM_TOKEN: ${{ secrets.HERMES_TELEGRAM_BOT_TOKEN }}
           TELEGRAM_CHANNEL_ID: ${{ secrets.TELEGRAM_OSSCAMEROON_CHANNEL_ID }}
+          consumer_key: ${{ secrets.TWITTER_CONSUMER_KEY }}
+          consumer_secret: ${{ secrets.TWITTER_CONSUMER_SECRET }}
+          api_key: ${{ secrets.TWITTER_API_KEY }}
+          api_secret: ${{ secrets.TWITTER_API_SECRET }}
```

Comment:
Why not keeping this as capital letters... ? 🤔

Reply:
I don't know, I just started like that for the other bots and I kept it that way ever since. I will change that 🙇‍♂️

Reply:
Done